scripts/gen_av1_chromium.py

```diff
@@ -0,0 +1,176 @@
+import sys
+import multiprocessing
+import urllib.request
+import json
+import re
+
+# pylint: disable=wrong-import-position
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+from fluster.test_suite import TestSuite, TestVector
+from fluster.codec import Codec, OutputFormat
+from fluster import utils
+from fluster.decoders import av1_aom
+
+# pylint: enable=wrong-import-position
+
+DOWNLOAD_URL = "https://storage.googleapis.com/chromiumos-test-assets-public/tast/cros/video/test_vectors/av1"
+
+TESTS_8BPP = (
+    # 8 bit
+    "00000527_20210205.ivf",
+    "00000535_20210205.ivf",
+    "00000548_20201006.ivf",
+    "48_delayed_20201006.ivf",
+    "av1-1-b8-02-allintra_20201006.ivf",
+    "av1-1-b8-03-sizeup_20201006.ivf",
+    "av1-1-b8-23-film_grain-50_20201006.ivf",
+    "ccvb_film_grain_20201006.ivf",
+    "frames_refs_short_signaling_20201006.ivf",
+    "non_uniform_tiling_20201006.ivf",
+    "test-25fps-192x288-only-tile-cols-is-power-of-2_20210111.ivf",
+    "test-25fps-192x288-only-tile-rows-is-power-of-2_20210111.ivf",
+    "test-25fps-192x288-tile-rows-3-tile-cols-3_20210111.ivf",
+)
+
+TESTS_10BPP = (
+    #10 bit
+    "00000671_20210310.ivf",
+    "00000672_20210310.ivf",
+    "00000673_20210310.ivf",
+    "00000674_20210310.ivf",
+    "00000675_20210310.ivf",
+    "00000716_20210310.ivf",
+    "00000717_20210310.ivf",
+    "00000718_20210310.ivf",
+    "00000719_20210310.ivf",
+    "00000720_20210310.ivf",
+    "00000761_20210310.ivf",
+    "00000762_20210310.ivf",
+    "00000763_20210310.ivf",
+    "00000764_20210310.ivf",
+    "00000765_20210310.ivf",
+    "av1-1-b10-00-quantizer-00_20210310.ivf",
+    "av1-1-b10-00-quantizer-10_20210310.ivf",
+    "av1-1-b10-00-quantizer-20_20210310.ivf",
+    "av1-1-b10-00-quantizer-30_20210310.ivf",
+    "av1-1-b10-00-quantizer-40_20210310.ivf",
+    "av1-1-b10-00-quantizer-50_20210310.ivf",
+    "av1-1-b10-00-quantizer-60_20210310.ivf",
+    "av1-1-b10-23-film_grain-50_20210310.ivf"
+)
+
+class ChromiumAV1Generator:
+    """Generates a test suite from the conformance bitstreams used in tast tests for Chromium"""
+
+    def __init__(
+        self,
+        name: str,
+        suite_name: str,
+        codec: Codec,
+        description: str,
+        bpp: int,
+    ):
+        self.name = name
+        self.suite_name = suite_name
+        self.codec = codec
+        self.description = description
+        self.decoder = av1_aom.AV1AOMDecoder()
+        self.bpp = bpp
+
+
+    def generate(self, download, jobs):
+        """Generates the test suite and saves it to a file"""
+        output_filepath = os.path.join(self.suite_name + ".json")
+        test_suite = TestSuite(
+            output_filepath,
+            "resources",
+            self.suite_name,
+            self.codec,
+            self.description,
+            dict(),
+        )
+
+        print(f"Download list of bitstreams from {DOWNLOAD_URL}")
+
+        TESTS = TESTS_10BPP if self.bpp == 10 else TESTS_8BPP
```

Comment:
I would suggest this change to make it more explicit and facilitate additional bit-depths in the future.

```suggestion
        if self.bpp == 10:
            TESTS = TESTS_10BPP 
        elif self.bpp == 8:
            TESTS = TESTS_8BPP
        else:
            return
```

Reply:
Excellent, thanks a lot!
I am not definitely not an expert in python, but as far as i know the smallest variable scope you have in python is a function. Everything defined within a function is accesible within the same.